config/default.schema.json

```diff
@@ -273,6 +273,11 @@
 			"type": "integer",
 			"default": 15,
 			"description": "Number of second the user will be blocked before logging in again"
+		},
+		"FEATURE_LERNSTORE_ENABLED": {
+			"type": "boolean",
+			"default": true,
```

Comment:
Default false, add default value false in default.json, add true in production.json finally for nbc-clients we have to set env to false